### DISCUSSION TITLE
30 day brew cleanup ends in error

### DISCUSSION THREAD

**Discussion by synthoidzeta** [Homebrew/discussions#777] [Casks]
`Error: Directory not empty @ dir_s_rmdir - /usr/local/Cellar/unbound/1.13.0`

I'm unclear how to resolve… this now occurs anytime I perform `brew upgrade`

**Comment by SMillerDev** [comment]
Try removing the directory manually

**Reply by charlesgreen** [reply]
I had the same problem and solved it with 

```
sudo chown -R <USER>:staff /usr/local/Cellar/unbound/1.13.0/
```

**Reply by apastuszak** [reply]
Thank you! This just helped me out a great deal.
